Xamarin.PropertyEditing.Mac/Controls/CharEditorControl.cs

```diff
@@ -13,7 +13,11 @@ public CharEditorControl (IHostResourceProvider hostResources)
 
 		protected override EntryPropertyEditorDelegate<char> CreateDelegate (PropertyViewModel<char> viewModel)
 		{
-			return new CharDelegate (viewModel);
+			var charDelegate = new CharDelegate (viewModel)
```

Comment:
Enforce ProxyResponder as a constructor parameter.

It will help catch places where the proxy responder isn't set by having it a required parameter.

Reply:
I want to do another review and improve the API, for the next update I will include the changes from your comments on it.